src/redux/features/deliverables/deliverablesThunks.ts

```diff
@@ -0,0 +1,36 @@
+import { Dispatch } from 'redux';
+import { RootState } from 'src/redux/rootReducer';
+import { SearchCriteria, SearchSortOrder } from 'src/types/Search';
+import DeliverablesService from 'src/services/DeliverablesService';
+import { setDeliverableListAction } from 'src/redux/features/deliverables/deliverablesSlice';
+
+export const requestDeliverableList = (
```

Comment:
use async thunks so we can leverage the status for ux purposes?

Reply:
I have another PR coming that converts these